### PR TITLE
Added Docker image badge & missing punctuation's

### PR DESCRIPTION
**Pull request by sumitkharche**
## Changes
- Added Docker Image badge
- Corrected grammar
- Added missing punctuation's

### CODE REVIEW

**Comment by knqyf263** [comment]
@sumitkharche I'm sorry, but would you resolve conflicts?

**Comment by sumitkharche** [comment]
> @sumitkharche I'm sorry, but would you resolve conflicts?

Yes, I will resolve the conflicts. Are the changes look good to you?

**Comment by sumitkharche** [comment]
@knqyf263 I have resolved the conflicts.

File: README.md [hunk 1]
```diff
@@ -1435,7 +1435,7 @@ $ trivy --reset
 
 - [Remic](https://github.com/aquasecurity/remic)
   - Vulnerability Scanner for Detecting Publicly Disclosed Vulnerabilities in Application Dependencies
-
+
```
Comment (knqyf263) [inline README.md]: Is this necessary?

Comment (sumitkharche) [reply README.md]: @knqyf263 No, It is added after resolving conflicts. I have removed the extra line.

**Pull request merged**